legacy.constraints

```diff
@@ -8,3 +8,5 @@ pluggy==0.5.2
 # will work with the above.
 pytest<3.7
 pytest-cov<2
+
+setuptools==11.2
```

Comment:
I feel unsure that this is actually working - I'm reading the log at https://github.com/release-engineering/pubtools/pull/24/checks?check_run_id=3347557618 but setuptools isn't shown there on the line where it dumps all the installed packages.  On the other hand, maybe it's not shown there due to slightly special nature of setuptools.

Do you know whether this actually works?  A way to prove it would be to see what happens if you add setuptools==11.2 into legacy.constraints but you don't include the bug fix here - if I understand right then that should trigger the bug, thus proving that CI can catch the bug.

If this doesn't actually work then I'd rather get rid of it.

Reply:
It works in my local testing. With setuptools==11.2 specified, I noticed another issue of ep.resolve() at loading pubtools.hooks entry points which I didn't fix in the original commit.

Reply:
OK, great.